content/en/beginner-install.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Beginner Installation Guide
+sidebar: false
+---
+
+{{< admonition tip >}}
+This is the beginner installation guide.
+If you are comfortable with using a terminal and happy to learn
+how to use a package manager, check out
+[the main installation guide](./install.md)!
+{{< /admonition >}}
+
+- [JupyterLite](#jupyterlite)
+- [Scientific Python Distributions](#distributions)
+- [Installing globally with `pip`](#pip-global)
+
+<a name="jupyterlite"></a>
+
+## JupyterLite
+
+To try out SciPy, you don't even need to install it!
+You can use SciPy in your browser at https://jupyter.org/try-jupyter/lab/ -
+just write `import scipy` in one of the notebook "cells" and hit play.
+
+For next steps, see [the SciPy user guide][scipy-user-guide].
+
+[scipy-user-guide]: https://docs.scipy.org/doc/scipy/tutorial/
+
+<a name="distributions"></a>
+
```

Comment:
Do they need any warnings here about losing their work if they clear their cache etc?

Reply:
not sure - suggestions welcome!